src/main/java/org/tron/core/config/Parameter.java

```diff
@@ -14,7 +14,8 @@
     double SOLIDIFIED_THRESHOLD = 0.7;
     int PRIVATE_KEY_LENGTH = 64;
     int MAX_ACTIVE_WITNESS_NUM = 27;
-    int TRXS_SIZE = 2_000_000; // < 2MiB
+    //int TRXS_SIZE = 2_000_000; // < 2MiB
+    int BLOCK_SIZE = 2_000_000;
     int BLOCK_PRODUCED_INTERVAL = 3000; //ms,produce block period, must be divisible by 60. millisecond
     long CLOCK_MAX_DELAY = 3600 * 1000; //ms
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Cast one of the operands of this multiplication operation to a "long". ([squid:S2184](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2184))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=833&issues=AWOcqCVZmXhJxGlVUHFI&open=AWOcqCVZmXhJxGlVUHFI)

[ISSUE_KEY]: AWOcqCVZmXhJxGlVUHFI